runtime/src/iree/hal/drivers/hip/hip_allocator.c

```diff
@@ -56,15 +51,19 @@ static iree_hal_hip_allocator_t* iree_hal_hip_allocator_cast(
 }
 
 iree_status_t iree_hal_hip_allocator_create(
-    const iree_hal_hip_dynamic_symbols_t* hip_symbols, hipDevice_t device,
-    hipCtx_t hip_context, hipStream_t stream,
+    const iree_hal_hip_dynamic_symbols_t* hip_symbols,
+    const iree_hal_hip_device_topology_t* topology,
     iree_hal_hip_memory_pools_t* pools, iree_allocator_t host_allocator,
     iree_hal_allocator_t** out_allocator) {
   IREE_ASSERT_ARGUMENT(hip_symbols);
   IREE_ASSERT_ARGUMENT(out_allocator);
   IREE_TRACE_ZONE_BEGIN(z0);
-  IREE_RETURN_AND_END_ZONE_IF_ERROR(
-      z0, iree_hal_hip_set_context(hip_symbols, hip_context));
+  *out_allocator = NULL;
+  if (topology->count < 1) {
+    IREE_TRACE_ZONE_END(z0);
+    return iree_make_status(IREE_STATUS_INVALID_ARGUMENT,
+                            "Invalid number of devices, must be at least one");
```

Comment:
```suggestion
                            "at least one device must be specified");
```

Reply:
```suggestion
                            "invalid number of devices at least one device must be specified");
```

(lower case, no trailing punctuation)